### PR TITLE
Inline shaders

### PR DESCRIPTION
**Pull request by IceDragon200**
#20

### CODE REVIEW

File: modules/engine/mrblib/engine.rb [hunk 1]
```diff
@@ -115,14 +115,12 @@ def reset_gl_flags
...
       @shaders = {}
-      shader_path = if Moon::Shader.is_legacy
-        'resources/shaders/120/'
-      else
-        'resources/shaders/330/'
-      end
-
-      @shaders[:quad] = Moon::Shader.load(shader_path + 'quad.vert', shader_path + 'quad.frag')
-      @shaders[:text] = Moon::Shader.load(shader_path + 'text.vert', shader_path + 'text.frag')
+      dfs = Moon::DEFAULT_SHADERS
```
Comment (archSeer) [inline modules/engine/mrblib/engine.rb]: Maybe, `Moon::Shader::DEFAULT(S)`?

File: modules/engine/mrblib/engine.rb [hunk 2]
```diff
@@ -115,14 +115,12 @@ def reset_gl_flags
...
+      shader_path = Moon::Shader.is_legacy ? '120/' : '330/'
+      @shaders[:quad] = Moon::Shader.new(dfs.fetch(shader_path + 'quad.vert'),
```
Comment (archSeer) [inline modules/engine/mrblib/engine.rb]: Might want to use `File.join` for joining filepaths, just in case.

Comment (IceDragon200) [inline modules/engine/mrblib/engine.rb]: mruby doesn't have one

**Comment by archSeer** [comment]
:+1: Now you get to update the tutorial yet again :3

**Pull request merged**